dags/hivemind_etl_helpers/src/db/common/base_preprocessor.py

```
@@ -0,0 +1,43 @@
+        text : str
+            a discord message text
+
+        Returns
+        --------
+        cleaned_text : str
+
+        """
+        try:
+            nlp = spacy.load("en_core_web_lg")
+        except OSError as exp:
+            raise OSError(f"Model spacy `en_core_web_lg` is not installed!") from exp
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)